### PR TITLE
Bin

### PR DESCRIPTION
**Pull request by vyadl**

### CODE REVIEW

File: src/components/sidebarContent/bin/AppBin.vue [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+<script>
+import { mapGetters, mapActions } from 'vuex';
+import SidebarCard from '@/components/wrappers/SidebarCard.vue';
+import CommonDeletedItem from '@/components/sidebarContent/bin/CommonDeletedItem.vue';
+import ButtonText from '@/components/formElements/ButtonText.vue';
+
+export default {
+  components: {
+    SidebarCard,
+    CommonDeletedItem,
```
Comment (moony-inc) [inline src/components/sidebarContent/bin/AppBin.vue:76]: renaming

File: src/components/sidebarContent/bin/AppBin.vue [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+    ButtonText,
+  },
+  data() {
+    return {
```
Comment (moony-inc) [inline src/components/sidebarContent/bin/AppBin.vue:80]: arrow function

File: src/components/sidebarContent/bin/CommonDeletedItem.vue [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    },
+  },
+  components: {
+    ButtonText,
+  },
+  methods: {
+    restore() {
+      this.$emit('restore');
+    },
+    remove() {
```
Comment (moony-inc) [inline src/components/sidebarContent/bin/CommonDeletedItem.vue:67]: why not "delete"?

File: src/store/actions.js [hunk 4]
```diff
@@ -184,9 +192,13 @@ export default {
       commit('setMode', 'list');
     }
   },
-  _setMode({ commit }, mode) {
+  _setMode({ commit, dispatch }, mode) {
     commit('setMode', mode);
     commit('setSorting', ['cloud', 'stars'].includes(mode) ? 'shuffled' : 'default');
+
+    if (mode === 'focus') {
```
Comment (moony-inc) [inline src/store/actions.js:199]: focus - isn't a mode 
it is in settings now

File: src/components/sidebarContent/bin/CommonDeletedItem.vue (hunk 3 above)
Comment (vyadl) [reply src/components/sidebarContent/bin/CommonDeletedItem.vue:67]: because vue think that delete in template is a method for deleting object property

**Pull request merged**